docs/application/tizen-studio/release-notes/3-3-release-notes.md

```diff
@@ -0,0 +1,66 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
+
+
+## IDE and Tools
+
+
+### Fixed Bugs
+-   Device Manager
+	-   Fixed the issue with logs not shown for Raspberry Pi board.
```

Comment:
To maintain consistency with other Release Notes (has/have been), please rephrase to:
The issue of logs not being displayed for the Raspberry Pi board has been fixed.

Please verify.